CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/prom2teams/tree/develop)
 
+## [2.1.3](https://github.com/idealista/prom2teams/tree/2.1.2)
```

Comment:
This will be a feature added so better change the minor version (2.2.0)

Reply:
Ops, my bad. Solve.